src/pages/Registration/types.ts

```diff
@@ -52,6 +52,7 @@ export type Documentation = {
   auditedFinancialReports: Asset;
   isKYCRequired: "Yes" | "No";
 
+  CashEligible: boolean;
```

Comment:
We should use lower case form field names for consistency.

Reply:
this was a request from paolo to align the attribute with DB

Reply:
It can be updated to lowercase for consistency and converted to uppercase in the HTTP request body

Reply:
@abbasali1994, `type Documentation` does not correspond to `AWS` but rather a local `type` for `<Form/>` usage - e.g. contains fields like `Asset` which is not present in any aws endpoints.  

`CashEligible` field should also be added in `type TDocumentation` in `types/aws/ap/registration` where it is consistent to AWS 

`cashEligible` can then be added to `type Documentation` in `pages/Registration/types`. I also recomment you use lowecase so to signal that this field is tied to `<Form/>` rather than `POST` endpoint

Reply:
Thnaks for the help,  @ap-justin will do this

Reply:
Done